<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemAddizionaliWidget</class>
 <widget class="QGroupBox" name="ItemAddizionaliWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Documenti legati alla moneta</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="itemsView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addItem">
       <property name="text">
        <string>Aggiungi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeItem">
       <property name="text">
        <string>Rimuovi</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
